### DISCUSSION TITLE
Is it pandasai 1.5.4 works with openai = 0.27.10?

### DISCUSSION THREAD

**Discussion by mraguth** [sinaptik-ai/pandas-ai#786] [Q&A] [answered]
If yes, how I install it? BTW I am using Azure Openai.

Thank you for your help.

**Comment by gventuri** [comment, marked as answer]
If I'm not mistaken, PandasAI now has upgraded openai to ^1.
Is there a specific reason why you need that version in particular? @mraguth

**Reply by mraguth** [reply]
If I upgrade my openai, I am getting some problems like.. 
ModuleNotFoundError: No module named 'openai.embeddings_utils'. I am not sure how I mitigate.

**Reply by mraguth** [reply]
Thank you. I am able to port new pandasai with Openai 1.1. Thank you for working on this.
